app/views/liquids/_open_graph.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="crayons-card c-embed text-styles text-styles--secondary">
+  <% if page.main_properties_present? %>
+    <% if page.image %>
+      <div class="c-embed__cover">
+        <a href="<%= page.url %>" class="c-link s:max-w-50 align-middle" target="_blank" rel="noopener noreferrer">
+          <img alt="" src="<%= page.image %>" height="auto" loading="lazy" class="m-0">
+        </a>
+      </div>
+    <% end %>
+    <div class="c-embed__body">
+      <h2 class="fs-xl lh-tight">
+        <a href="<%= page.url %>" target="_blank" rel="noopener noreferrer" class="c-link">
+          <%= page.title %>
+        </a>
+      </h2>
+      <% if page.description %>
+        <p class="truncate-at-3">
+          <%= page.description %>
```

Comment:
this `page.description` seems to be the place in the image that we're not unescaping html entities, I suspect that's the case for title as well (sample size of 1 didn't have a title with an html entity in it).

Reply:
I assume there's a better named view helper for this step - but `Nokogiri::HTML.fragment` seems to do the right thing in this case. I tested with embedded tags, for example if you have a script literal in the description meta, the tag content gets spliced in while the tag itself disappears. I'm assume we're more concerned with properly handling innocent data, and defanging malicious data, than with beautifying bad data.  

```ruby
page = OpenGraph.new('https://takeonrules.com/about/')

page.description
=> "Jeremy Friesen&rsquo;s personal blog, with a focus on table-top role-playing games."

Nokogiri::HTML.fragment(page.description)
=> #(DocumentFragment:0x25ad0 { name = "#document-fragment", children = [ #(Text "Jeremy Friesen's personal blog, with a focus on table-top role-playing games.")] })

Nokogiri::HTML.fragment(page.description).content
=> "Jeremy Friesen’s personal blog, with a focus on table-top role-playing games."


# now with tags
problem_description = '</div></span><script>alert("explode");</script>Jeremy&rsquo;s Blog'
=> "</div></span><script>alert(\"explode\");</script>Jeremy&rsquo;s Blog"
Nokogiri::HTML.fragment(problem_description).content
=> "alert(\"explode\");Jeremy’s Blog"
```

Reply:
Oooh, I like the idea of removing the tags entirely, but still leaving their content. Makes it effectively work like `innerText`, which is likely the behavior we want here.

<img width="277" alt="Example of creating an HTML element, setting the innerHTML property, and then reading the innerText property. The HTML tags are removed, but their content remains, with HTML entities unescaped." src="https://user-images.githubusercontent.com/108205/161592750-adcec03b-bfa7-4f95-bde6-07d4081ca3a1.png">